client/wfnews-war/src/main/angular/src/app/components/full-details/area-restrictions-full-details/area-restrictions-full-details.component.ts

```diff
@@ -0,0 +1,145 @@
+    super(router);
+    this.restrictionID = fullDetails.restrictionID;
+    this.agolService = agolService;
+  }
+
+ ngOnInit(): void {
+  this.populateAreaRestrictionByID(this.restrictionID)
+  this.populateIncident()
+  this.populateUrls()
+ } 
+
+ initMap(latitude: string, longitude: string): void {
+    this.cdr.detectChanges()
+
+    // Create map and append data to the map component
+    const location = [Number(latitude), Number(longitude)]
+
+    this.map = L.map('map', {
+    attributionControl: false,
+    zoomControl: false,
+    dragging: false,
+    doubleClickZoom: false,
+    boxZoom: false,
+    trackResize: false,
+    scrollWheelZoom: false,
+    }).setView(location, 9)
+    // configure map data
+    L.tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
+      attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+    }).addTo(this.map)
+
+    const databcUrl = this.appConfigService.getConfig()['mapServices']['openmapsBaseUrl'].toString()
+    L.tileLayer.wms(databcUrl, {
+      layers: 'WHSE_LAND_AND_NATURAL_RESOURCE.PROT_RESTRICTED_AREAS_SP ',
+      format: 'image/png',
+      transparent: true,
+      version: '1.1.1',
+      opacity: 0.5
+    }).addTo(this.map)
+
+    const fireOfNoteIcon = L.icon({
+      iconUrl: "/assets/images/local_fire_department.png",
+      iconSize: [35, 35],
+      shadowAnchor: [4, 62],
+      popupAnchor: [1, -34],
+      shadowSize: [41, 41]
+    });
+
+    if(this.incident.fireOfNoteInd) {
+      L.marker(location, {icon: fireOfNoteIcon}).addTo(this.map);
+    }else{
+      let colorToDisplay;
+      switch(this.incident.stageOfControlCode) {
+        case 'OUT_CNTRL':
+          colorToDisplay = '#FF0000'
+          break;
+        case 'HOLDING':
+          colorToDisplay = '#ffff00'
+          break;
+        case 'UNDR_CNTRL':
+          colorToDisplay = '#98E600'
+          break;
+        case 'OUT':
+          colorToDisplay = '#999999'
+          break;
+        default:
+          colorToDisplay = 'white';
+      }
+      L.circleMarker(location,{radius:15,fillOpacity:1,color:'black', fillColor:colorToDisplay}).addTo(this.map)
+    }
+    this.cdr.detectChanges()
+  }
+
+ populateAreaRestrictionByID(id: string) {
+    let areaRestrict = this.agolService.getAreaRestrictionsByID(id).toPromise().then((response) => {
+      if (response.features && response.features[0] && response.features[0].attributes) {
+          let name: string = response.features[0].attributes.NAME;
+          name = name.replace("Area Restriction", "").trim();
+          this.name = name + " Restricted Area"
+          this.fireCentre = response.features[0].attributes.FIRE_CENTRE_NAME + " Fire Centre";
+          this.issuedDate = convertToDateYear(response.features[0].attributes.ACCESS_STATUS_EFFECTIVE_DATE);
+      }
+   })
+ }
+
+ populateIncident() {
+  // Using these values as a stub until preview page is complete to pass the incident number from. 
```

Comment:
Based on our conversation, looks like we need to run a spatial query with area restrictions polygon to check overlap incidents. And once we get that, it can calls populateIncident with incident number if there is a result.